tests/core/full_node/test_mempool.py

```diff
@@ -311,8 +282,10 @@ def test_fun(coin_1: Coin, coin_2: Coin) -> SpendBundle:
             bundle = SpendBundle.aggregate([tx1, tx2])
             return bundle
 
-        full_node_1, _, server_1, _ = two_nodes_mempool
-        blocks, bundle, status, err = await self.condition_tester2(bt, two_nodes_mempool, wallet_a, test_fun)
+        full_node_1, server_1 = one_node_one_block
+        _ = await next_block(full_node_1, wallet_a, bt)
+        _ = await next_block(full_node_1, wallet_a, bt)
```

Comment:
oh,. I see. you just make two extra blocks here to make the numbers the same as before. you would get less code and faster tests by just decrementing the expected block heights by 2 instead

Reply:
It wasn't passing though